test/remote/gateways/remote_forte_test.rb

```diff
@@ -23,6 +23,9 @@ def setup
       billing_address: address,
       description: 'Store Purchase'
     }
+
+    @options_with_order_id = @options
+    @options_with_order_id.update(order_id:'2')
```

Comment:
Is there any reason to not send an `order_id` with all transactions? Most test suites include the `order_id` in the default `options` hash.